standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2537,19 +2583,21 @@ service ThriftHiveMetastore extends fb303.FacebookService
   void drop_catalog(1: DropCatalogRequest catName) throws (1:NoSuchObjectException o1, 2:InvalidOperationException o2, 3:MetaException o3)
 
   void create_database(1:Database database) throws(1:AlreadyExistsException o1, 2:InvalidObjectException o2, 3:MetaException o3)
+  void create_database_req(1:CreateDatabaseRequest createDatabaseRequest) throws(1:AlreadyExistsException o1, 2:InvalidObjectException o2, 3:MetaException o3)
```

Comment:
can we rename the method to `create_database` to be more generic? same to other methods.

Reply:
all the request based apis should have a suffix `_req()` as it is with other methods in hive_metastore.thrift